docs/user-guide/debugging-services.md

```diff
@@ -256,6 +256,14 @@ If this fails, you might need to go to the kube-proxy section of this doc, or
 even go back to the top of this document and start over, but instead of
 debugging your own `Service`, debug DNS.
 
+### _Some_ of my images cannot resolve
+
+Check if the container image can resolve DNS via `search`.
+
+For example, Alpine Linux uses `musl-libc` and does not support the `search`
+keyword in `resolv.conf`. If you want to use Alpine as base image you will need
+to get an specially built for Kubernetes.
```

Comment:
If you want to use Alpine as a base image you will need a special build that works with Kubernetes.